package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "redash-client",
-  "version": "24.05.0-dev",
+  "version": "24.04.0-dev",
```

Comment:
This one should probably be the `05` version here, as it's a continuation of the 05 series now. :smile: